### DISCUSSION TITLE
Does Jan support being a frontend for a llama.cpp/koboldcpp server?

### DISCUSSION THREAD

**Discussion by InconsolableCellist** [janhq/jan#3398] [Get Help] [answered]
I already have a server but the Koboldcpp Lite UI is a bit limited, and I'd like something more OpenAI-like. The Jan UI seems like an improvement for chat purposes. Can it connect to a remote backend like llamacpp, koboldcpp, or an OpenAI compatible API?

**Comment by ux-han** [comment, marked as answer]
Currently no, but there's feature request here, we'll be tracking it: https://github.com/janhq/jan/issues/2998
